backend/lib/services/terminals/terminalBootstrap.js

```diff
@@ -346,9 +349,21 @@ async function ensureTrustedCertForShootApiServer (client, shootResource, seedRe
     return
   }
 
-  if (!seedResource.spec.secretRef) {
-    logger.info(`Bootstrapping Shoot ${namespace}/${name} aborted as 'spec.secretRef' on the seed is missing. In case a shoot is used as seed, add the flag \`with-secret-ref\` to the \`shoot.gardener.cloud/use-as-seed\` annotation`)
-    return
+  let seedClient
+
+  const managedSeed = await getManagedSeedOrUndefined(client, seedName)
+  if (managedSeed) {
+    const shootRef = {
```

Comment:
This is also used above. May be extract a helper function:
```js
function getShootRef (managedSeed) {
   return {
      namespace: managedSeed.metadata.namespace,
      name: managedSeed.spec.shoot.name
    }
}
```

Reply:
✅